cms/startup.py

```diff
@@ -25,6 +26,9 @@ def run():
 
     add_mimetypes()
 
+    if settings.FEATURES.get('ENABLE_NOTIFICATIONS', False):
+        startup_notification_subsystem()
```

Comment:
Are there any return codes or error states that should be caught/communicated at this level of this startup process?